controllers/__snapshots__/aliases.test.js.snap

```diff
@@ -2212,83 +2212,83 @@ Array [
   },
   Object {
     "from": "/funding/awards-for-all",
-    "to": "/funding/under10k",
+    "to": "/funding/under-10k",
```

Comment:
Any reason we're changing this slug? Assuming we'll need to add a redirect from `funding/under10k` to handle the change>